src/protocols/secure_channel/PASESession.h

```diff
@@ -76,10 +76,24 @@ struct PASESessionSerializable
     uint16_t mPeerSessionId;
 };
 
+typedef uint8_t PASEVerifierSerialized[2 * kSpake2p_WS_Length];
```

Comment:
what will be the use for it?
Please specify a format. If this is to be persisted to disk, should we have a header at least (like 4 bytes out of which some are version like {'S' 'P' 'V' 1} or something. Then deserialize can actually validate input data instead of just copying.

Reply:
Added detailed description.
Some of the use cases: serialized output of the PBKDF that generated verifier. when verifier is sent over the wire, and for storage. Adding version to this serialized structure doesn't seem appropriate.

Reply:
So why is a header not needed for serialized data?

All serialized data generally has a header (e.g. elf header, gif header, etc.) and all over the wire data has a header (e.g. tcp/udp themsevles have header and version, CHIP messages have headers and so on). Why is this structure special? Convince me that the space savings (generally 4 bytes) are worth it and we do not need error checks.

Reply:
@andy31415 Spec currently defines PASE Verifier as a raw `(W0, L)` data without header.

Also this implementation is similar to how data is serialized in other places in the SDK:
`PASESessionSerializable`
`P256SerializedKeypair`

If we want to change this serialization approach - I think that should be done as a separate work/PR.